delete.go

```diff
@@ -39,7 +42,41 @@ EXAMPLE:
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return delete(context.String("container-id"), context.Bool("force"))
 	},
 }
+
+func delete(containerID string, force bool) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkDeleteParamsIntegrity(containerID); err != nil {
+		return err
+	}
+
+	if _, err := vc.StopPod(containerID); err != nil {
+		return err
+	}
+
```

Comment:
@sameo I think I have a solution to know if the container is running. We just need to check if the shim PID still exists, right ?
In case we can find it, this means it is not stopped and we can return an error in case we don't have "force" enabled.
How does that sound ?